README.md

```diff
@@ -43,17 +40,49 @@ Using the command line:
   - Invoke "Test" target from "Build.proj". Likely, you need to build test project first, so put in "build" target as well. 
         *msbuild build.proj /t:build;test /p:scope=ResourceManagement\Compute*
 
-### To on-board new libraries
-Follow existing library and create a new folder under "ResourceManagement".
-  - Note: To simplify test discovery, the test folder must be named with ".test" or ".tests"
-  
-If for platform reasons that your library won't use NetCore project system, 3 notes
-  - In your library csproject file, set the msbuild property "AutoRestProjects" to "true"
-  - In your test project files, set both "AutoRestProjects" and "SDKTestProject" to "true"
-  - To simplify test discovery, the test folder must be named with ".tests"
+## To on-board new libraries
+
+### Project Structure
+
+In "src\ResourceManagement", you will find packages for services that have already been implemented
+
+  - Each service contains a package for their generated code, as well as a package for tests
+  - The file 'generate.cmd', used to generate library code for the given service, can also be found in these packages
+
+### Branches: AutoRest vs. master
+
+The **AutoRest** branch contains the code generated from AutoRest.
+
+The **master** branch contains the code generated from Hydra/Hyak.
+  - Hydra/Hyak is Azure's legacy code generation technology.
+  - This can still be used to generate client libraries, but the project is not being advanced in favor of AutoRest. Your team should move to AutoRest and Swagger as soon as possible.
+
+### Standard Process
+
+For a step-by-step process on how to generate C# code for your service and get it published, click [here](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/autorest-to-powershell.md#phase-1---swagger-specs-to-net-sdk-using-autorest).
+
+**Note**: if you receive a 404 error trying to access this page, click [here](https://aka.ms/azuregithub) and sign in with your Microsoft corporate account to gain access.
+
+### Code Review Process
+
+To see the requirements and process for checking code into the working branch, click [here](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_sdk_gitflow.md#checking-code-into-the-dev-working-branch)
+
+**Note**: if you receive a 404 error trying to access this page, click [here](https://aka.ms/azuregithub) and sign in with your Microsoft corporate account to gain access.
+
+### Adding Tests
+
+Regarding the test project, one thing that's important is to name the test project folder and related project assembly to be ".tests".
```

Comment:
Name the test project by adding a ".Tests" suffix to the product assembly name.  For example, the test project for 'Microsoft.Azure.Management.Compute' should be named 'Microsoft.Azure.Management.Compute.Tests'